dbfit-java/hsqldb/src/main/java/dbfit/util/sql/HSQLDBPreparedStatements.java

```diff
@@ -0,0 +1,8 @@
+package dbfit.util.sql;
+
+public class HSQLDBPreparedStatements extends PreparedStatements {
+
+    public static String buildFunctionCall(String procName, int numberOfParameters) {
+        return "{ call " + procName + "(" + buildParamList(numberOfParameters - 1) + ")}";
```

Comment:
This looks pretty much like `buildStoredProcedureCall` with reduced number of parameters. What about the option to rather use `buildStoredProcedureCall` directly instead of introducing this child HSQLDBPreparedStatements?

PreparedStatements used to be a simple bunch of helpers. Possibly their names were not the best. Would it be better to have `buildCallWithReturnValue` and `builtCallWithoutReturnValue`?